easybuild/easyconfigs/l/libevent/libevent-2.0.22-GCC-4.9.2.eb

```diff
@@ -0,0 +1,17 @@
+easyblock = 'ConfigureMake'
+
+name = 'libevent'
+version = '2.0.22'
+
+homepage = 'http://libevent.org/'
+description = """The libevent API provides a mechanism to execute a callback function when a specific
+ event occurs on a file descriptor or after a timeout has been reached.
+ Furthermore, libevent also support callbacks due to signals or regular timeouts."""
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
+
+source_urls = ['https://github.com/downloads/libevent/libevent/']
+sources = ['%(name)s-%(version)s-stable.tar.gz']
+source_urls = ['https://sourceforge.net/projects/levent/files/%(name)s/%(name)s-2.0/']
```

Comment:
Agreed, the former no longer works, hence the second, and I forgot to remove the first.

@boegel  With respect to keys such as 'name', 'version', and friends: it would be very useful to have an exhaustive list somewhere in the documentation.  Perhaps it is there already, and I missed it?